### PR TITLE
Implement blogs for codewind

### PR DESCRIPTION
**Pull request by sghung**
Implement blogs for codewind and put medium codewind content onto there.

Notable tests:
- I tested the re-size on smaller resolutions too. The blog posts should be minimized correctly, although the images may not get minimized if a width was added to it
- I clicked on various external (e.g. npm links) and internal links (codewind links) in the blog
- I currently don't generate a trailing .html for blog posts. I think you need to do something else to generate that. Since medium doesn't use .html, I don't think we need to either

One extremely frustrating thing is in enabling-https-in-your-codewind-application. If you try to put the xml formatting to it, it doesn't work. It is because of the numbered/unordered list, it doesn't work very well. You would need to have an empty line and then the xml content if you want it to show up. This is inconsistent with other formatting, so I had to manually copy the HTML and insert it in there.

### CODE REVIEW

File: docs/_blog/2019-10-30-a-technical-autopsy-of-a-containerized-node-js-dependency-insights-microservice-application.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: "A technical autopsy of a containerized ‘Node.js dependency insights’ microservice application"
+
```
Comment (sishida) [inline docs/_blog/2019-10-30-a-technical-autopsy-of-a-containerized-node-js-dependency-insights-microservice-application.md]: Is there a reason a space is here? Will it cause a problem?

File: docs/_blog/2019-10-30-checking-node-js-sub-dependencies-licenses-for-usage-and-redistribution.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+### Pain point 3: Compile an accurate understanding of sub-dependency licenses and copyright notices
+Again this is not quite as easy as it sounds:
+1. The good news for Node.js is that the community is good at declaring the license for a module in the package.json. I used a ‘[license-checker](https://www.npmjs.com/package/license-checker){:target="_blank"}’ module on NPM, which does a great job reporting the declared license and in the absence of a declaration, it parses targeted files and guesses the license.
+1. However, it’s possible that code under less commercially redistribution friendly licenses lurks in package files, so it is advantageous to check all files for ‘risky keywords’ such as ‘GPL’.
+1. Some packages release under ‘dual’ license terms, so you need to do your due diligence to ensure that you can use a package under the accepted license terms for your offering.
+1. Each of the cases above can require manual investigation, so rediscovery of risks and issues can result in wasted time, especially when packages with trickier cases release frequently.
```
Comment (sishida) [inline docs/_blog/2019-10-30-checking-node-js-sub-dependencies-licenses-for-usage-and-redistribution.md]: All of the numbers on this blog a 1. Do they autoformat to become sequential, or is this a typo?

Comment (sghung) [reply docs/_blog/2019-10-30-checking-node-js-sub-dependencies-licenses-for-usage-and-redistribution.md]: The auto format will number it properly. On other posts, I did the manual way, I think this is an earlier one I converted

Comment (sghung) [reply docs/_blog/2019-10-30-checking-node-js-sub-dependencies-licenses-for-usage-and-redistribution.md]: To make it consistent, I can just type out the numbers manually

File: docs/_blog/2019-10-30-a-technical-autopsy-of-a-containerized-node-js-dependency-insights-microservice-application.md (hunk 1 above)
Comment (sghung) [reply docs/_blog/2019-10-30-a-technical-autopsy-of-a-containerized-node-js-dependency-insights-microservice-application.md]: I removed the space, no it won't cause a problem, but to be consistent I will remove it

**Pull request merged**